src/compiler/clang.rs

```diff
@@ -16,7 +16,7 @@
 
 use crate::compiler::args::*;
 use crate::compiler::c::{
-    ArtifactDesciptor, CCompilerImpl, CCompilerKind, Language, ParsedArguments,
+    ArtifactDescriptor, CCompilerImpl, CCompilerKind, Language, ParsedArguments,
```

Comment:
Yikes, that was merged recently. Could we extract this changeset aloing with the `derive(Eq)`  one into a separate PR?